stage6/01-install-packages/00-packages

```diff
@@ -87,3 +87,4 @@ dialog
 ckermit
 software-properties-common
 xserver-xorg-video-dummy
+idle3
```

Comment:
is this a dependency ? Wouldn't this be brought automatically when the package is installed?

Reply:
Unfortunately, even with the original apt packages, this dependency is missing. Issue is seen during the execution of example scripts from the GUI